libnmstate/schema.py

```diff
@@ -227,12 +227,14 @@ class Port(object):
         VLAN_SUBTREE = 'vlan'
 
         class Vlan(object):
-            TRUNK_TAGS = 'trunk-tags'
-            TAG = 'tag'
             ENABLE_NATIVE = 'enable-native'
-            TYPE = 'type'
-            ACCESS_TYPE = 'access'
-            TRUNK_TYPE = 'trunk'
+            MODE = 'type'
```

Comment:
Why don't you change the right-hand-side too?

Reply:
Missing in this PR a few points.
Will treat it in a few hours.
Thanks for the review.

Reply:
Done